src/DrupalInitCommand.php

```diff
@@ -140,6 +140,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $options['config'] = [
                 'sort-packages' => true,
                 'optimize-autoloader' => true,
+                'apcu-autoloader' => true,
+                'secure-http' => false,
```

Comment:
@softeg, I am not sure if this is a good idea to turn off by default. I know there are cases where `secure-http` must be disabled but I would rather letting the user do this rather than set it for them.